plugins/modules/eos_static_routes.py

```diff
@@ -848,11 +591,45 @@
 #                next_hop:
 #                  - interface: "Ethernet1"
 
-# returns:
 
+# Task Output
+# -------------
+# rendered:
+# - ip route 165.10.1.0/24 Ethernet1 10.1.1.2 100
+# - ipv6 route 5001::/64 Ethernet1
+
+
+# Using parsed:
+
+
+# parse_static_routes.cfg
 # ip route 165.10.1.0/24 Ethernet1 10.1.1.2 100
 # ipv6 route 5001::/64 Ethernet1
+#
+
+
+- name: parse configs
+  arista.eos.eos_static_routes:
+    running_config: "{{ lookup('file', './parse_static_routes.cfg') }}"
+    state: parsed
+
 
+# Task Output
+# -------------
+# parsed:
+# - address_families:
+#   - afi: ipv4
+#     routes:
+#     - dest: 165.10.1.0/24
+#       next_hop:
+#       - forward_router_address: 10.1.1.2
+#         interface: Ethernet1
+#         admin_distance: 100
+#   - afi: ipv6
+#     routes:
+#     - dest: 5001::/64
+#       next_hop:
+#       - interface: Ethernet1
```

Comment:
RETURN SECTION needs to be updated for left-out operations,
example: https://github.com/ansible-collections/junipernetworks.junos/blob/6f75fdad3f0a2d7dc3378b22f58943f26f406fc2/plugins/modules/junos_bgp_address_family.py#L1954